server/lib/tuist_web/live/qa_run_live.html.heex

```diff
@@ -0,0 +1,240 @@
+          >
+            <div data-part="root">
+              <div data-part="trigger" data-state="closed">
+                <div data-part="issue-icon">
+                  <.icon name="alert_hexagon" />
+                </div>
+                <div data-part="issue-title">{gettext("Temporary visual inconsistency")}</div>
+                <div data-part="issue-toggle">
+                  <.neutral_button
+                    data-part="closed-collapsible-button"
+                    variant="secondary"
+                    size="medium"
+                  >
+                    <.chevron_down />
+                  </.neutral_button>
+                  <.neutral_button
+                    data-part="open-collapsible-button"
+                    variant="secondary"
+                    size="medium"
+                  >
+                    <.chevron_up />
+                  </.neutral_button>
+                </div>
+              </div>
+              <div data-part="content" data-state="closed">
+                <div data-part="issue-content-wrapper">
+                  <div :if={issue_data.screenshot} data-part="screenshots-column">
+                    <div data-part="screenshot-thumb">
+                      <img
+                        src={
+                          ~p"/#{@qa_run.app_build.preview.project.account.name}/#{@qa_run.app_build.preview.project.name}/qa/runs/#{@qa_run.id}/screenshots/#{issue_data.screenshot.id}"
+                        }
+                        alt="Screenshot"
+                      />
+                    </div>
+                  </div>
+
+                  <div data-part="step-details">
+                    <div data-part="step-detail">
+                      <.badge label="Action" color="neutral" style="light-fill" size="small" />
+                      <span data-part="detail-value">
+                        {raw(Earmark.as_html!(issue_data.step.action))}
+                      </span>
+                    </div>
+
+                    <div data-part="step-detail">
+                      <.badge label="Result" color="information" style="light-fill" size="small" />
+                      <span data-part="detail-value" data-variant="success">
+                        {raw(Earmark.as_html!(issue_data.step.result))}
+                      </span>
+                    </div>
+
+                    <div data-part="step-detail">
+                      <.badge label="Issue" color="warning" style="light-fill" size="small" />
+                      <span data-part="detail-value" data-variant="warning">
+                        {raw(Earmark.as_html!(issue_data.issue))}
+                      </span>
+                    </div>
+
+                    <div data-part="step-detail">
+                      <.badge label="Timestamp" color="focus" style="light-fill" size="small" />
+                      <span data-part="detail-value">
+                        {DateFormatter.format_full(issue_data.step.inserted_at)}
```

Comment:
Similar to the "Duration", I think this should be a timestamp of the recording (i.e. from the point we run the app on the simulator). Let's keep this as-is, I can update this along with the timeline